seedlet-blocks/sass/blocks/_links.scss

```diff
@@ -15,7 +15,7 @@
  */
 .is-root-container, // Editor class hack.
 .wp-block-post-content {
-	a {
+	a:not(.wp-block-button__link) {
```

Comment:
This change was only tangentially related. The anchor style was causing the button (anchor) styles to be black instead of the configured white.